crates/rspack_core/src/stats.rs

```diff
@@ -136,13 +137,26 @@ impl Stats<'_> {
     nested_modules: bool,
     source: bool,
   ) -> Result<Vec<StatsModule>> {
-    let mut modules: Vec<StatsModule> = self
+    let modules: Vec<StatsModule> = self
       .compilation
       .module_graph
       .modules()
       .values()
       .map(|module| self.get_module(module, reasons, module_assets, nested_modules, source))
       .collect::<Result<_>>()?;
+
+    let runtime_modules: Vec<StatsModule> = self
+      .compilation
+      .runtime_modules
+      .iter()
+      .map(|(identifier, module)| {
+        self.get_runtime_module(identifier, module, reasons, module_assets)
+      })
+      .collect::<Result<_>>()?;
+    let mut modules = modules
+      .into_iter()
+      .chain(runtime_modules.into_iter())
+      .collect::<Vec<_>>();
```

Comment:
I think this can be done isomorphically by line 140, if module's module_type is `ModuleType::Runtime`, then its stats.moduleType will also be "runtime", so the correct solution should be adding `ModuleType::Runtime` to runtime module

Reply:
We can add `ModuleType::Runtime` to runtime module in ` impl_runtime_module` marco

Reply:
>  done isomorphically by line 140
Does this means we should combine `get_module` and `get_runtime_module` into one function?

Reply:
Oh, I see, then current implementation looks good to me